File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
    let maxcount=0;
    for(let i=0; i<sentences.length; i++) {
            maxcount = Math.max(maxcount,(sentences[i].split(" ")).length);
        }
        return maxcount;
    
};

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int maxcount=0;
        for(int i=0; i<sentences.length; i++) {
            maxcount = Math.max(maxcount,(sentences[i].split(" ")).length);
        }
        return maxcount;
    }
}